src/test/resources/local-pipeline/run_local_pipeline

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+echo "Checking args"
+if [ $# -ne 5 ];
+then
+	echo "Execution using 5 (five) arguments:"
+	echo "arg1 - usgs username."
+	echo "arg2 - usgs password."
+	echo "arg3 - image dataset in format landsat_X, where X (landsat number)."
+	echo "arg4 - image region in format PPPRRR, where P (path) and R (row)."
+	echo "arg5 - image date in format YYYY-MM-DD, where Y (year), M (month) and D (day)."
+	exit 1
+fi
+USGS_USERNAME=$1
+USGS_PASSWORD=$2
+IMAGE_DATASET=$3
+IMAGE_REGION=$4
+IMAGE_DATE=$5
+
+echo "Checking docker"
+which docker &> log
```

Comment:
I think it is nice to check docker install. kudos!

However, I think it is better to NOT help the user installing it. Just `exit 1`